patches/minecraft/net/minecraft/client/model/HumanoidModel.java.patch

```diff
@@ -0,0 +1,195 @@
+--- a/net/minecraft/client/model/HumanoidModel.java
++++ b/net/minecraft/client/model/HumanoidModel.java
+@@ -32,8 +_,8 @@
+    public final ModelPart f_102812_;
+    public final ModelPart f_102813_;
+    public final ModelPart f_102814_;
+-   public HumanoidModel.ArmPose f_102815_ = HumanoidModel.ArmPose.EMPTY;
+-   public HumanoidModel.ArmPose f_102816_ = HumanoidModel.ArmPose.EMPTY;
++   public net.minecraftforge.common.extensions.IForgeArmPose f_102815_ = HumanoidModel.ArmPose.EMPTY;
++   public net.minecraftforge.common.extensions.IForgeArmPose f_102816_ = HumanoidModel.ArmPose.EMPTY;
+    public boolean f_102817_;
+    public float f_102818_;
+ 
+@@ -142,7 +_,7 @@
+             this.m_102878_(p_102866_);
+          }
+       } else {
+-         boolean flag4 = flag2 ? this.f_102815_.m_102897_() : this.f_102816_.m_102897_();
++         boolean flag4 = flag2 ? this.f_102815_.isPoseTwoHanded() : this.f_102816_.isPoseTwoHanded();
+          if (flag2 != flag4) {
+             this.m_102878_(p_102866_);
+             this.m_102875_(p_102866_);
+@@ -227,73 +_,85 @@
+    }
+ 
+    private void m_102875_(T p_102876_) {
+-      switch(this.f_102816_) {
```

Comment:
This patch could be a lot smaller.
If `ArmPos` is extensible like suggested above, just add a `default` block that calls `applyTransform`.